user-operator/src/test/java/io/strimzi/operator/user/model/KafkaUserModelTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.user.model;
+
+import io.strimzi.api.kafka.model.KafkaUser;
+import io.strimzi.api.kafka.model.KafkaUserSpec;
+import io.strimzi.api.kafka.model.KafkaUserTlsClientAuthentication;
+import io.strimzi.operator.common.model.Labels;
+import io.strimzi.operator.common.operator.MockCertManager;
+import io.strimzi.operator.user.ResourceUtils;
+
+import java.util.Base64;
+
+import io.fabric8.kubernetes.api.model.Secret;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class KafkaUserModelTest {
+    private final KafkaUser user = ResourceUtils.createKafkaUser();
+    private final Secret clientsCa = ResourceUtils.createClientsCa();
+    private final Secret userCert = ResourceUtils.createUserCert();
+
+    @Test
+    public void testFromCrd()   {
+        KafkaUserModel model = KafkaUserModel.fromCrd(new MockCertManager(), user, clientsCa, null);
```

Comment:
I would make `MockCertManager` a field (thus only instantiated once rather than _N_ times, which helps if we ever had to add ctor parameters to it).